openshift/deploy/job/cas-shelf-tfe-add-app.yml

```diff
@@ -0,0 +1,35 @@
+
+apiVersion: batch/v1
```

Comment:
```suggestion
objects
  - apiVersion: batch/v1
```
and everything below needs to be indented one level

Reply:
you should be able to run `make lint` (after being logged in pathfinder) to ensure that this file is well formed